kpi/tests/api/v1/test_api_imports.py

```diff
@@ -74,6 +74,39 @@ def test_import_asset_base64_xls(self):
         self._post_import_task_and_compare_created_asset_to_source(task_data,
                                                                    self.asset)
 
+    def test_import_asset_library(self):
+       library_xls = open('../../../fixtures/basic-text-library.xlsx', 'rb').read()
```

Comment:
as discussed, i'll demonstrate a way to construct the xls within the test from an easy-to-read python structure